content/en/docs/tasks/traffic-management/ingress/secure-ingress/index.md

```diff
@@ -336,43 +491,90 @@ retrieves unique credentials corresponding to a specific `credentialName`.
 
 ### Configure a mutual TLS ingress gateway
 
-You can extend your gateway's definition to support
-[mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication). Change
-the credentials of the ingress gateway by deleting its secret and creating a new one.
-The server uses the CA certificate to verify
-its clients, and we must use the name `cacert` to hold the CA certificate.
+You can extend your gateway's definition to support [mutual TLS](https://en.wikipedia.org/wiki/Mutual_authentication).
+
+1. Change the credentials of the ingress gateway by deleting its secret and creating a new one.
+    The server uses the CA certificate to verify its clients, and we must use the name `cacert` to hold the CA certificate.
+
+    {{< text bash >}}
+    $ kubectl -n istio-system delete secret httpbin-credential
+    $ kubectl create -n istio-system secret generic httpbin-credential \
```

Comment:
very nit, outside of scope: instead of delete+create which has downtime, I wonder if we should do `kubectl create --dry-run | kubectl apply`